hypothesis-python/src/hypothesis/core.py

```diff
@@ -894,6 +894,25 @@ def fake_subTest(self, msg=None, **__):
     yield
 
 
+@attr.s()
+class HypothesisHandle(object):
```

Comment:
Not required as part of this PR, but it would be nice to move the other attributes we put on a test function into here too. Maybe we should open an issue about that? Despite the general horror of core.py it might even be a good first issue.

Reply:
Unless we have a functional reason to move them, I'd leave the attributes where they are - [Hyrum's Law](http://www.hyrumslaw.com/) suggests that (a) someone will have broken code if we move them, but (b) I also don't want to put them next to a documented thing which would people more likely to use them.